workspace.bzl

```diff
@@ -1,13 +1,13 @@
 load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
 load("@blackjack//:cargo_dependencies.bzl", "cargo_dependencies")
 
-def blackjack_cargo():
+def blackjack_cargo(target="x86_64-unknown-linux-gnu", sha256="ac2746e3d3bab7301b8aa747eff7c4d66f9c88a61f9117a4d6669c40317b69cc"):
```

Comment:
Ideally I think this would work similar to `rules_rust` toolchain resolution, with defaults provided for multiple platforms: https://github.com/bazelbuild/rules_rust/blob/master/rust/repositories.bzl#L87.

Maybe you could define a few `http_archive`s for linux x64 and any others you would like to support, and then create an alias with a [`select()`](https://docs.bazel.build/versions/master/configurable-attributes.html)?

Something like:
```
http_archive(name = "blackjack_cargo_x86_64-unknown-linux-gnu", ... )

http_archive(name = "blackjack_cargo_x86_64-apple-darwin", ... )

alias(
  name = "blackjack_cargo",
  actual = select({
    "@io_bazel_rules_rust//rust/platform:x86_64-unknown-linux-gnu": ["@blackjack_cargo_x86_64-unknown-linux-gnu//:cargo"],
    ...
  }),
)
```

We can also leave this for now, blackjack will also work with a locally installed cargo, so this isn't paramount for multi-platform support.

Reply:
Ah, I didn't know that a locally installed cargo works as well! We can split this into two PRs then!